client/src/pages/gmailAuth/AuthRoute.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import axios from 'axios'
+import queryString from 'query-string'
+import withAuth from "common/withAuth";
+import useSnackbar from 'common/useSnackbar';
+
+// A Route that responds when OAuth redirects the user back to our application
+
+const Auth = (props) => {
+
+    const showSnackBar = useSnackbar()
```

Comment:
Are you using prettier on the front-end - noticed things are missing like ; for example